packages/api/src/utility.ts

```diff
@@ -4,6 +4,81 @@ import {delegateToSchema, IDelegateToSchemaOptions, Transform, ExecutionResult}
 
 import {Context} from './context'
 import {TeaserStyle} from './db/block'
+import {User} from './db/user'
+
+export function mapSubscriptionsAsCsv(items: User[]) {
+  const JsonFields = [
+    'id',
+    'name',
+    'email',
+    'active',
+    'createdAt',
+    'modifiedAt',
+
+    'company',
+    'streetAddress',
+    'streetAddress2',
+    'zipCode',
+    'city',
+    'country',
+
+    'memberPlanID',
+    'paymentPeriodicity',
+    'monthlyAmount',
+    'autoRenew',
+    'startsAt',
+    'paidUntil',
+    'paymentMethodID',
+    'deactivatedAt'
+  ]
+
+  let csvStr = JsonFields.join(',') + '\n'
+
+  items.forEach(({address, subscription, ...user}: any) => {
+    csvStr +=
+      user.id +
+      ',' +
+      user.name +
+      ',' +
+      user.email +
+      ',' +
+      user.active +
+      ',' +
+      new Date(user.createdAt).toISOString() +
+      ',' +
+      new Date(user.modifiedAt).toISOString() +
+      ',' +
+      (address?.company || '') +
+      ',' +
+      (address?.streetAddress || '') +
+      ',' +
+      (address?.streetAddress2 || '') +
+      ',' +
+      (address?.zipCode || '') +
+      ',' +
+      (address?.city || '') +
+      ',' +
+      (address?.country || '') +
+      ',' +
+      subscription!.memberPlanID +
+      ',' +
+      subscription!.paymentPeriodicity +
+      ',' +
+      subscription!.monthlyAmount +
+      ',' +
+      subscription!.autoRenew +
+      ',' +
+      new Date(subscription!.startsAt).toISOString() +
+      ',' +
+      (subscription?.paidUntil || '') +
+      ',' +
+      subscription!.paymentMethodID +
+      ',' +
+      new Date(subscription!.deactivatedAt).toISOString() +
+      '\r\n'
```

Comment:
For readability I would go:
```
[value, value, value].join(',') + '/r/n'
```
as with `jsonFields`

Reply:
@tomaszdurka  please re-check.